src/index.html

```diff
@@ -1,12 +1,385 @@
+              </a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#Gadgets" class="header__nav-link">Gadgets</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#Tablets" class="header__nav-link">Tablets</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#Photo" class="header__nav-link">Photo</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#Video" class="header__nav-link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </section>
+    <section class="page__cards">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Product image"
+          class="card__image"
+        >
+
+        <div class="card__description">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviwes-value">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">
+            Price:
+          </p>
+          <p class="card__value">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__linc"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Product image"
+          class="card__image"
+        >
+
+        <div class="card__description">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviwes-value">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">
+            Price:
+          </p>
+          <p class="card__value">
+            $2,199
+          </p>
+        </div>
+
+        <a href="#"class="card__linc">buy</a>
+      </div>
+      <div class="card">
```

Comment:
Add a blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```